<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoDialog</class>
 <widget class="QDialog" name="InfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wellcome to QImageViewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/pict/QImageViewer.png</normaloff>:/pict/QImageViewer.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="6" column="0">
    <widget class="QPushButton" name="openBtn">
     <property name="text">
      <string> Open (Ctrl+O)</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/pict/open.png</normaloff>:/pict/open.png</iconset>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/pict/QImageViewer.png&quot;/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'DejaVu Sans Mono'; font-size:14pt; font-weight:600; color:#008000;&quot;&gt;QImageViewer&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-style:italic; color:#555753;&quot;&gt;Version 1.0.0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="saveBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string> Save (Ctrl+S)</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/pict/save.png</normaloff>:/pict/save.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QPushButton" name="rotLeftBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string> Rotate (Ctrl+Left)  </string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/pict/rotate_l.png</normaloff>:/pict/rotate_l.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="2">
    <widget class="QPushButton" name="aboutQtBtn">
     <property name="text">
      <string> About Qt (F2)</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/pict/help.png</normaloff>:/pict/help.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="printBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string> Print (Ctrl+P)</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/pict/print.png</normaloff>:/pict/print.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="14" column="2">
    <widget class="QPushButton" name="rotRightBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string> Rotate (Ctrl+Right)</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/pict/rotate_r.png</normaloff>:/pict/rotate_r.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="aboutBtn">
     <property name="text">
      <string> About (F1)</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/pict/info.png</normaloff>:/pict/info.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QPushButton" name="zoomInBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string> Zoom In (Ctrl++)</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/pict/zoomIn.png</normaloff>:/pict/zoomIn.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="fullScrBtn">
     <property name="text">
      <string> Fullscreen (F11)</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/pict/full.png</normaloff>:/pict/full.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="delBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string> Delete (Del)</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/pict/trash.png</normaloff>:/pict/trash.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="0">
    <widget class="QPushButton" name="prevBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string> Previous (Left &lt;-)</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/pict/left.png</normaloff>:/pict/left.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QPushButton" name="nextBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string> Next (Right -&gt;)       </string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/pict/right.png</normaloff>:/pict/right.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QPushButton" name="zoomOutBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string> Zoom out (ctrl+-)</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/pict/zoomOut.png</normaloff>:/pict/zoomOut.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
